README.md

```diff
@@ -1 +1,50 @@
 # HangOps Code of Conduct
+
+To continue growing a safe, welcoming, and fair community, all HangOps members are required to comply with the following code of conduct. HangOps admins are responsible for ensuring that all community members follow this code within HangOps spaces.
+
+## The Short Version
+
+Be respectful of other people; respectfully ask people to stop if you are bothered; and if you can’t resolve an issue contact the admins. If you do not follow our Community Code of Conduct, it will be apparent and you may be removed from HangOps.
```

Comment:
Thanks! For now I've gone with:

> respectfully ask people to stop if you feel disrespected

which I think encompasses many potential scenarios without being overly prescriptive about the sort of actions which may be deemed "offensive".